### ISSUE TITLE
Improved Delete Account

### ISSUE THREAD

**Issue by Pdzly** [sublinks/sublinks-api#345]

**Comment by jgrim** [comment]
There needs to be consideration for accounts without an email set. Magic links won't always be an option.

Is it 2fa if enabled and Password if no 2fa?

**Comment by Pdzly** [comment]
@jgrim yeah, i edited it, and yeah if 2fa is available 2fa then ONLY use 2fa, if no 2fa is active then password.

But no "workaround" if he loses 2fa and wants to delete his account.
